packages/mds-types/index.ts

```diff
@@ -453,6 +453,10 @@ export interface AuditDetails extends Audit {
   }
 }
 
+export interface PolicyMessage {
```

Comment:
Seems like a kinda weird type to define, but even if we want to, it doesn't need to be an exported type for sure

Reply:
Well the spec does say it has to be of the form `{ 'language_code': 'message' }`, so might as well, but I can def remove the `export`